From db6ed9ede3a3ad5f8ff69e0e2cdb9872ae0f7775 Mon Sep 17 00:00:00 2001
From: Mario Limonciello <mario.limonciello@dell.com>
Date: Fri, 26 Jan 2018 10:35:06 -0600
Subject: [PATCH] data/installed_tests: Redirect stderr to stdout for one of
 the tests

The autopkgtest test cases in Debian and Ubuntu are failing due to
output present in stderr.

fwupdmgr behavior however is that it's expected that this test should
show output on stderr for this item.
---
 data/installed-tests/fwupdmgr.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/data/installed-tests/fwupdmgr.sh b/data/installed-tests/fwupdmgr.sh
index da69d58..72147ba 100755
--- a/data/installed-tests/fwupdmgr.sh
+++ b/data/installed-tests/fwupdmgr.sh
@@ -64,7 +64,7 @@ rc=$?; if [[ $rc != 0 ]]; then exit $rc; fi
 
 # ---
 echo "Getting updates (should be none)..."
-fwupdmgr --no-unreported-check --no-metadata-check get-updates
+fwupdmgr --no-unreported-check --no-metadata-check get-updates 2>&1
 rc=$?; if [[ $rc != 2 ]]; then exit $rc; fi
 
 # ---
-- 
2.7.4

